docs/pages/blog/material-ui-v7-is-out.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Material UI v7 is out 🚀
+description: Material UI v7 is now stable, improving integration with modern tools and consistency across the library.
+date: 2025-03-31T12:00:00.000Z
+authors: ['diegoandai']
+tags: ['Material UI', 'Product']
+manualCard: true
+---
+
+Material UI v7 is now stable.
+This major improves the integration with modern tools and consistency across the library.
+It's a lean upgrade designed to be straightforward to upgrade to.
+
+## Improved ESM support
+
+The package layout has been updated to comply with ESM.
+This is achieved by adding the `exports` field to the `package.json` file.
+The `export` field indicates where bundlers can find CJS, ESM, and modern bundle files.
+You can read more about this in the [Node.js documentation](https://nodejs.org/api/packages.html#packages_exports).
+
+This update fixes several issues with popular bundlers like Vite and webpack.
+It also reduces the amount of setup required to work with these tools.
+
+## Opt-in support for CSS layers
+
+The `enableCssLayer` prop has been added to `ThemeProvider`.
+When enabled, Material UI styles are wrapped in a CSS layer named `@mui`.
+
+This allows Material UI to integrate with modern tools that rely on CSS layers, like Tailwind CSS v4.
```

Comment:
It's the non-breaking space instead of regular space - here to avoid breaking the word over two lines. 😉 
```suggestion
This allows Material UI to integrate with modern tools that rely on CSS layers, like Tailwind CSS v4.
```